### DISCUSSION TITLE
2023-03-29 Weekly Meeting 101 Agenda/Notes

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#373] [Weekly Meetings]
Weekly Meeting 101 is happening on Wednesday, March 29, 2023 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
... placeholder

**Comment by madjin** [comment]
## Vket Exhibitor Application Open

- https://event.vket.com/en/docs/exhibition_guideline

![image](https://user-images.githubusercontent.com/32600939/228600404-2f7b6e0c-0288-4599-b59c-314865d891c9.png)

We made a bunch of booths already, can reuse and update some of them

![image](https://user-images.githubusercontent.com/32600939/228600688-55ab1635-b640-42d8-9857-7b472b92d9cb.png)

This can be a great way to bring together other metaverse interop orgs / organizations / projects like those in MSF to build a metaverse expo in the metaverse. This can be a great way for people to see the landscape, learn about what each other is working on, and network.

![image](https://user-images.githubusercontent.com/32600939/228600612-90703eae-abb0-4e06-8f32-8fcd520c8fc0.png)


![image](https://user-images.githubusercontent.com/32600939/228601402-3cd923c5-fb06-47c5-b777-2530e627cc13.png)

**Comment by madjin** [comment]
## GDC Updates

- Spatial audio convos / networking
- OMI meetup :D 
- Cool AI hackathon / art event (Beamable / CultureDAO)
- Tilt5 table was noteworthy, impressed by UGC content
- Web3 had bigger presence (polygon / immutable news)
- Tim Sweeney talk worth rewatching https://www.youtube.com/watch?v=gcDLDiMBGao
  - UEFN https://dev.epicgames.com/documentation/en-us/uefn/unreal-editor-for-fortnite-documentation
- Some folks hiring avatar artists: Futureway

**Comment by madjin** [comment]
### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
  - shifting to 501c, applying for a grant (Epic Mega Grants), MSF-update itself 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - update draft protocol submission #356 
  - omi_personality
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [UX Research](https://github.com/omigroup/ux-research) 
- [OMI Website update](https://github.com/omigroup/website)
- any others?

## MSF Delegates

- Digital Asset Managment presentation tomorrow
- MSF oversight meeting last Wednesday
  - Moving towards final phases of incorporation (bylaws / membership structures finalized)
  - EOM moving into this more official structure
  - We just have to confirm our membership
    - OMI currently principle member, we fall into special SDO category (open source standards org, meets no dues criteria)
    - We have to read bylaws, just have to sign new membership agreement, have couple months tho
- There's an initial board, temporary - official elections later, we may want official representation
  - OMI can represent community / grassroots interests
  - Need to have membership agreement signed asap if so
- Field trip to MVFW (Metaverse Fashion Week) tomorrow
  - https://twitter.com/monaverse/status/1640745957938675713
  - https://twitter.com/search?q=decentraland%20mvfw&src=typed_query

##  glTF Extensions

- May present tomorrow at the Digital Asset Management WG (MSF)

## Media Group

- Did a rehearsal last week in podcast studio
- Content production opportunity tomorrow during MVFW

## M3 Avatar Group

- New blog post on Metadata / Avatar creator program: https://mirror.xyz/m3org.eth/4NPYOzRtcuYHrRI7iZK1yADswgOv9To2ErQ8KrqLPX0

## UX research

Would like to participate in defining metaverse, need time / schedule, could align with field trip?

Many OMI folks speaking at AWE (end of May), sharing research with public - good milestone

## OMI Website

We should add MSF logo - jin
![MSF_Principal_RGB_170px_July22](https://user-images.githubusercontent.com/32600939/228608380-4a0779a2-70b2-49a9-a225-d127136be34b.png)
